os/iphone-default-sdk.cmake

```diff
@@ -0,0 +1,67 @@
+# This script sets the following variables :
+# IOS_SDK_VERSION : will contain the version number of the default iOS SDK (example : 11.0)
+# IPHONEOS_SDK_ROOT : full path to the SDK
+# IPHONEOS_ROOT
+# XCODE_DEVELOPER_ROOT
+
+if(DEFINED POLLY_IPHONE_DEFAULT_SDK_CMAKE)
+  return()
+else()
+  set(POLLY_IPHONE_DEFAULT_SDK_CMAKE 1)
+endif()
+
+include("${CMAKE_CURRENT_LIST_DIR}/../utilities/polly_status_debug.cmake")
+
+# polly_find_xcode_ios_defaults : 
+# fills 
+# * XCODE_DEVELOPER_ROOT
+# * IPHONEOS_ROOT 
+# * IPHONEOS_SDK_ROOT
+# * IOS_SDK_VERSION
+macro (polly_find_xcode_ios_defaults)
+  find_program(XCODE_SELECT_EXECUTABLE xcode-select)
+  if(NOT XCODE_SELECT_EXECUTABLE)
+    polly_fatal_error("xcode-select not found")
+  endif()
+  execute_process(
+    COMMAND
+    ${XCODE_SELECT_EXECUTABLE}
+    "-print-path"
+    OUTPUT_VARIABLE
+    XCODE_DEVELOPER_ROOT # /.../Xcode.app/Contents/Developer
+    OUTPUT_STRIP_TRAILING_WHITESPACE
+  )
+  set(IPHONEOS_ROOT "${XCODE_DEVELOPER_ROOT}/Platforms/iPhoneOS.platform/Developer")
+  # The defautl SDK is at ${IPHONEOS_ROOT}/SDKs/iPhoneOS.sdk
+  set(IPHONEOS_SDK_ROOT "${IPHONEOS_ROOT}/SDKs/iPhoneOS.sdk")
+  polly_status_debug("XCODE_DEVELOPER_ROOT=${XCODE_DEVELOPER_ROOT}")
+  polly_status_debug("IPHONEOS_ROOT=${IPHONEOS_ROOT}")
+  polly_status_debug("IPHONEOS_SDK_ROOT=${IPHONEOS_SDK_ROOT}")
+endmacro()
+
+macro (polly_find_iphone_sdk_version)
```

Comment:
Do we need to wrap this part with macro?

Reply:
It's a matter of taste : they could be reused elsewhere.
Anyhow,I removed the second macro